pkgs/development/python-modules/nf-core/default.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  click,
+  filetype,
+  gitpython,
+  jinja2,
+  jsonschema,
+  markdown,
+  packaging,
+  pdiff,
+  pillow,
+  pre-commit,
+  prompt-toolkit,
+  pytest,
+  pytest-workflow,
+  pyyaml,
+  questionary,
+  # NOTE Not going to support as it would require packaging up 10+ additional
+  # packages and is a niche feature nf-core plans on dropping support for
+  # refgenie
```

Comment:
This comment should be just before `pythonRemoveDeps`.
We won't know that something is missing until it is explicitly removed.